server/apps/organization/management/commands/init_organizations.py

```diff
@@ -0,0 +1,69 @@
+import requests
+
+from django.core.management.base import BaseCommand
+
+from apps.organization.models import OrganizationKind, Organization
+
+
+class Command(BaseCommand):
+    help = 'Initialize Organizations'
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.org_kinds = dict()
+
+    def fetch_organizations(self, offset=0, limit=1000):
+        print('Fetching organizations starting from: {} to {}'.format(offset, offset + limit))
+        URL = 'https://api.reliefweb.int/v1/sources?fields[include][]=logo&fields[include][]=' \
+              'country.iso3&fields[include][]=shortname&fields[include][]=longname&fields' \
+              '[include][]=homepage&fields[include][]=type&offset={}&limit={}'.format(
+                offset,
+                limit,
+                )
```

Comment:
Let's try to use multi-line with bracket. and use f-string. I think this looks cleaner and easy to understand. What do you think?
```suggestion
        URL = (
            'https://api.reliefweb.int/v1/sources?fields[include][]=logo&fields[include][]='
            'country.iso3&fields[include][]=shortname&fields[include][]=longname&fields'
            f'[include][]=homepage&fields[include][]=type&offset={offset}&limit={limit}'
        )
```